app/page.tsx

```diff
@@ -1,7 +1,34 @@
-export default function Index(){
-    return(
-        <h1>
-            Stellar Wallet
-        </h1>
-    )
-}
\ No newline at end of file
+export default function Index() {
```

Comment:
This has to be typed as a component.

Reply:
I have already committed this change and now the Index has the React functional component type